packages/twenty-front/src/utils/image/getImageAbsoluteURI.ts

```diff
@@ -1,15 +1,26 @@
 import { REACT_APP_SERVER_BASE_URL } from '~/config';
 
-export const getImageAbsoluteURI = (imageUrl?: string | null) => {
+type ImageAbsoluteURI<T extends string | null | undefined> = T extends string
```

Comment:
this feels odd, not sure what issue it was trying to solve but we can probably do better no?